src/mapping/tests/GinkgoRadialBasisFctSolverTest.cpp

```diff
@@ -50,72 +50,72 @@ BOOST_AUTO_TEST_SUITE(GinkgoRadialBasisFunctionSolver)
 #define TEST_FOR_ALL_RBFS(EXECUTOR, SOLVER)                                              \
```

Comment:
You can do this directly in [boost test](https://www.boost.org/doc/libs/1_85_0/libs/test/doc/html/boost_test/tests_organization/test_cases/test_organization_templates.html#boost_test.tests_organization.test_cases.test_organization_templates.test_case_template_with_manual_r)
For another PR though